infra.csv

```diff
@@ -46,6 +52,6 @@ shfmt,Tools,Adopt,TRUE,
 Shellcheck,Tools,Adopt,TRUE,
 Kustomize,Tools,Adopt,FALSE,"<p><a href=""https://github.com/kubernetes-sigs/kustomize"">Kustomize</a> is a YAML templating tool to render k8s resources. Used in <a href=""https://github.com/camunda/infra-core"">infra-core</a> project.</p>"
 config-lint,Tools,Adopt,FALSE,"<p><a href=""https://github.com/stelligent/config-lint"">config-lint</a> is a tool to validate configuration files using rules specified in YAML. We're use it locally and in CI pipelines to lint and ensure Terraform best practices like <a href=""https://www.terraform.io/docs/extend/best-practices/naming.html"">naming convention</a>.</p>"
-Atlantis,Tools,Assess,TRUE,
```

Comment:
maybe move it to `HOLD` instead?

Reply:
@maxdanilov but we never used it afaik ... it was under evaluation but never used.